R/content.R

```diff
@@ -1587,21 +1584,40 @@ get_content_packages <- function(content) {
 search_content <- function(
   client,
   q = NULL,
-  page_number = 1,
-  page_size = 500,
   include = "owner,vanity_url",
   ...
 ) {
   error_if_less_than(client$version, "2024.04.0")
 
-  path <- v1_url("search", "content")
+  inner_search <- function(
```

Comment:
I know I called it this in my quick snippet, but what about this: 

```suggestion
  .search_content <- function(
```

Reply:
Sure!